website/content/en/docs/golang/quickstart.md

```diff
@@ -248,7 +248,7 @@ You can set the `Result.RequeueAfter` to requeue the `Request` after a grace per
 ```Go
 import "time"
 
-// Reconcile for any reason than error after 5 seconds
+// Reconcile for any reason other than error after 5 seconds
```

Comment:
```suggestion
// Reconcile for any reason other than an error after 5 seconds
```

Reply:
I was reusing the wording from the above section (reque immediately/without time). Do you think that section should be changed to "other than an error" also?

Reply:
Sure